internal/cluster/cluster.go

```diff
@@ -777,10 +778,28 @@ func (m *Manager) DeleteClusterFiles(ctx context.Context, c *common.Cluster, obj
 	return nil
 }
 
-func (m Manager) PermanentClustersDeletion(ctx context.Context, olderThen strfmt.DateTime, objectHandler s3wrapper.API) error {
+func (m Manager) InactiveClusterDeregister(ctx context.Context, inactiveSince strfmt.DateTime, objectHandler s3wrapper.API) error {
 	var clusters []*common.Cluster
 	db := m.db.Unscoped()
-	if reply := db.Where("deleted_at < ?", olderThen).Find(&clusters); reply.Error != nil {
+	if reply := db.Where("updated_at < ?", inactiveSince).Find(&clusters); reply.Error != nil {
+		return reply.Error
+	}
+	for i := range clusters {
+		c := clusters[i]
+		m.eventsHandler.AddEvent(ctx, *c.ID, nil, models.EventSeverityError,
```

Comment:
Don't think that it should be with severity error - I think it should be a warning or info. @avishayt WDYT?

Reply:
Agree

Reply:
+1 info, users will not see those events anyway it's for our use.